packages/smart-wallet/src/offerWatcher.js

```diff
@@ -274,7 +274,20 @@ export const prepareOfferWatcher = (baggage, vowTools) => {
       resultWatcher: {
         onFulfilled(result) {
           const { facets } = this;
-          facets.helper.publishResult(result);
+          const { walletHelper } = this.state;
+          const { after } = this.state.status;
+          if (after) {
+            if (after.saveAs) {
+              const { saveAs: name } = after;
+              // Note: result is passable and storable since it was received from another vat
+              walletHelper.saveOfferResult(name, result);
+              facets.helper.publishResult(
```

Comment:
this is no longer publishing the result.

why even post to vstorage here?

Reply:
I think having details of the outcome of the offer is still valuable. The outcome in this case was to save the result.